content/_code-samples/use-tickets/py/use-tickets.py

```diff
@@ -0,0 +1,54 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models.transactions import TicketCreate, AccountSet
+from xrpl.transaction import safe_sign_and_submit_transaction
+from xrpl.wallet import Wallet, generate_faucet_wallet
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+
+# Connect to a testnet node
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Generate a wallet and request faucet
+test_wallet = generate_faucet_wallet(client=client)
+myAddr = test_wallet.classic_address
+
+# Construct a TicketCreate transaction, 1 ticket created for future use
```

Comment:
Can you create multiple tickets and show how to pick between them? (I think the main use will be to send multiple transactions, so it's important to highlight the step where you can distribute more than one ticket sequence)

Reply:
(Even just 2 is enough)

Reply:
Implemented.